pkg/volume/volume.go

```diff
@@ -77,6 +77,10 @@ type Attributes struct {
 type Mounter interface {
 	// Uses Interface to provide the path for Docker binds.
 	Volume
+
+	//CanMount returns if the volume can be mounted
+	CanMount() bool
```

Comment:
Instead of returning a bool, how about returning an error. So that if the check fails plugins can return an error with a message for what exactly is missing. e.g. `Required binary /sbin/mount.nfs4 is missing.` The caller, operation executor, can then use this message in the user visible error to provide more actionable information.

(Update the comment accordingly).